<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="buttonBrowse">
          <property name="text">
           <string>Обзор</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxBrowseColor">
          <item>
           <property name="text">
            <string>Черный</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Красный</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Зеленый</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Синий</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="lineSearch"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QPushButton" name="buttonSearch">
          <property name="text">
           <string>Поиск</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxSearchColor">
          <item>
           <property name="text">
            <string>Черный</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Красный</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Зеленый</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Синий</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="CustomTextBrowser" name="textBrowser">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Название:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineName"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Жанр:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxFilmGenre">
        <item>
         <property name="text">
          <string>Романтика</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Триллер</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Фантастика</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Экшен</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Год выхода:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineYearRelease"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPushButton" name="buttonAddFig">
          <property name="text">
           <string>Добавить</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxAddFigColor">
          <item>
           <property name="text">
            <string>Черный</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Красный</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Зеленый</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Синий</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLineEdit" name="lineLhs"/>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxCompare">
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>=</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>&lt;</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>&gt;</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineRhs"/>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>?</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineComparison">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>не знаю</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomTextBrowser</class>
   <extends>QTextBrowser</extends>
   <header location="global">customtextbrowser.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBrowse</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>browse()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>133</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonSearch</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>search()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>788</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>712</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonAddFig</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>add_film()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>788</x>
     <y>504</y>
    </hint>
    <hint type="destinationlabel">
     <x>703</x>
     <y>572</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBoxCompare</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>checkYears()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>215</x>
     <y>554</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>484</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineRhs</sender>
   <signal>editingFinished()</signal>
   <receiver>MainWindow</receiver>
   <slot>checkYears()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>289</x>
     <y>553</y>
    </hint>
    <hint type="destinationlabel">
     <x>261</x>
     <y>538</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineLhs</sender>
   <signal>editingFinished()</signal>
   <receiver>MainWindow</receiver>
   <slot>checkYears()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>561</y>
    </hint>
    <hint type="destinationlabel">
     <x>110</x>
     <y>531</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>browse()</slot>
  <slot>search()</slot>
  <slot>magic()</slot>
  <slot>yearSort()</slot>
  <slot>add_fig()</slot>
  <slot>add_film()</slot>
  <slot>checkYears()</slot>
 </slots>
</ui>
